src/ZArray.jl

```diff
@@ -333,6 +333,14 @@ function zcreate(::Type{T},storage::AbstractStore,
   attrs=Dict(),
   writeable=true,
   ) where T
+
+  if compressor isa AbstractString
+    if haskey(compressortypes, String(compressor))
+      compressor = compressortypes[compressor]()
```

Comment:
This would make it impossible to set custom compression levels for the compression algorithm. Do we need another keyword argument for `zcreate` that gets passed to the compressor constructor?

Reply:
The idea here is that the simple option of just passing a string will give you default compression options. If you want to specify the compression level, you can use the compression constructor and pass the instatiated compressor instance.

Reply:
i get and error when trying to pass a compression constructor:

```
julia> zzeros(Int16, 16, 16, compressor=CodecZstd.ZstdCompressor(; level=6), path=tempname())
ERROR: Cannot serialize type Ptr{CodecZstd.LibZstd.ZSTD_CCtx_s}
```